src/pages/CreatePool/index.tsx

```diff
@@ -127,36 +167,43 @@ export default function CreatePool(): React.ReactElement {
               <Typography mb={2}>{t("createPoolTypeDescription2")}</Typography>
               <Typography mb={2}>{t("createPoolTypeDescription3")}</Typography>
               <ToggleButtonGroup
-                value="basePool"
+                value={poolType}
                 color="secondary"
                 size="large"
+                exclusive
+                onChange={(event, value) => setPoolType(value)}
                 fullWidth
               >
-                <ToggleButton value="usdMetaPool" size="large">
+                <ToggleButton value="usdMetapool" size="large">
                   USD Metapool
                 </ToggleButton>
-                <ToggleButton value="btcMetapool">USD Metapool</ToggleButton>
-                <ToggleButton value="basePool">USD Metapool</ToggleButton>
+                <ToggleButton value="btcMetapool">BTC Metapool</ToggleButton>
+                <ToggleButton value="baseMetapool">Base Metapool</ToggleButton>
               </ToggleButtonGroup>
             </Box>
-            <Box flex={1}>
+            <Box
+              flex={1}
+              display={poolType === "baseMetapool" ? "block" : "none"}
```

Comment:
Instead of using display block here you should leverage `flex` and it should automatically adjust to take entire "row". The above comment should help with accomplishing that.

If the above doesn't work giving the `<Stack>` the `flexWrap: 'wrap'` rule might make it take full width.